tests/backup/backup_kubevirt_test.go

```diff
@@ -3288,3 +3290,404 @@ var _ = Describe("{CustomBackupRestoreWithKubevirtAndNonKubevirtNS}", Label(Test
 	})
 })
+
+// This testcase verifies backup & restore during Kubevirt VM migration.
+var _ = Describe("{KubevirtVMMigrationTest}", Label(TestCaseLabelsMap[KubevirtVMMigrationTestLabel]...), func() {
+	var (
+		backupNames          = make([]string, 0)
+		sourceClusterUid     string
+		cloudCredName        string
+		cloudCredUID         string
+		backupLocationUID    string
+		backupLocationName   string
+		providers            []string
+		restoreNames         []string
+		backupLocationMap    = make(map[string]string)
+		backupNamespaceMap   = make(map[string]string)
+		labelSelectors       = make(map[string]string)
+		wg                   sync.WaitGroup
+		mu                   sync.RWMutex
+		scheduledAppContexts []*scheduler.Context
+		allVMs               []kubevirtv1.VirtualMachine
+		allVMNames           []string
+		freezeRuleName       string
+		unfreezeRuleName     string
+		freezeRuleUid        string
+		unfreezeRuleUid      string
+	)
+
+	JustBeforeEach(func() {
+		StartPxBackupTorpedoTest("KubevirtVMMigrationTest", "verifies backup & restore during Kubevirt VM migration.", nil, 93442, Ak, Q1FY25)
+		providers = GetBackupProviders()
+		log.InfoD("scheduling applications")
+		scheduledAppContexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%d-%d", 93442, i)
+			appContexts := ScheduleApplications(taskName)
+			for _, appCtx := range appContexts {
+				appCtx.ReadinessTimeout = AppReadinessTimeout
+				scheduledAppContexts = append(scheduledAppContexts, appCtx)
+			}
+		}
+	})
+
+	It("Verify backup & restore during Kubevirt VM migration", func() {
+		defer func() {
+			log.InfoD("Switching to default context")
+			err := SetClusterContext("")
+			log.FailOnError(err, "Failed to set ClusterContext to default cluster")
+		}()
+
+		Step("Validate applications", func() {
+			log.InfoD("Validating applications")
+			ValidateApplications(scheduledAppContexts)
+		})
+
+		Step("Creating backup location and cloud setting", func() {
+			log.InfoD("Creating backup location and cloud setting")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, provider := range providers {
+				cloudCredName = fmt.Sprintf("%s-%s-%s", "cred", provider, RandomString(6))
+				backupLocationName = fmt.Sprintf("%s-%v", getGlobalBucketName(provider), RandomString(6))
+				cloudCredUID = uuid.New()
+				backupLocationUID = uuid.New()
+				backupLocationMap[backupLocationUID] = backupLocationName
+				err := CreateCloudCredential(provider, cloudCredName, cloudCredUID, BackupOrgID, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of cloud credential named [%s] for org [%s] with [%s] as provider", cloudCredName, BackupOrgID, provider))
+				err = CreateBackupLocation(provider, backupLocationName, backupLocationUID, cloudCredName, cloudCredUID, getGlobalBucketName(provider), BackupOrgID, "", true)
```

Comment:
Dont set this to true. False is the default. Let it be false.

Reply:
Removed rule creation, and set this to false.